javascript/core/sparkplug-payload/lib/SparkplugPayload.proto

```diff
@@ -0,0 +1,125 @@
+package org.eclipse.tahu.protobuf;
```

Comment:
Does this file really need to be duplicated here?

Reply:
ah! didn't realize there was an existing proto file.